project_euler/problem_107/sol1.c

```diff
@@ -0,0 +1,188 @@
+/**
+ * \file
+ * \brief Project Euler problem 107 (https://projecteuler.net/problem=107)
+ * solution using Kruskals
+ * \author [David Santamaria](https://github.com/Daves1245)
+ */
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
+
```

Comment:
```suggestion

/**
 * @addtogroup project_euler_107 Project Euler 107
 * @{
 */
```
Since there are many associated functions and variables involved, better to group them for easy navigation on documentation site